ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -80,6 +80,7 @@ descriptor:
   exercise: "Mode of running equipment to maintain functionality ('exercise mode')"
   exhaust: "Process or component used in the removal of air from a conditioned system to the outside atmosphere."
   fault: "Fault of fire alarm system/panel e.g. ground fault, sensor fault, etc. A common of fire alarm system. It will not induce alarm."
+  panel: "An interface to control a device."
```

Comment:
What about electrical panels? Those aren't used for control. 

The usage of panel in `control_panel_mode` is not helpful in distinguishing it from `control_mode`; therefore it should be changed.

Reply:
We have been looking at the list of subfields to create a new name. However, we could find a suitable combination for this point. The point is for local panel selector switch to switch between Remote or Local. Could you help to suggest a name for this?

![image](https://user-images.githubusercontent.com/80947411/124637481-608cf300-debc-11eb-943a-4cad3ae32b2c.png)

Reply:
They are different mode. The control_mode with the state of AUTO or MANUAL and it is software point. However, the control_panel_mode is the local panel selector switch with the state of REMOTE, LOCAL or OFF, and it is hardware mode.

Reply:
changed to control_status

Reply:
control_panel_mode changed to control_status